src/gam/gapi/chromemanagement.py

```diff
@@ -217,6 +217,10 @@ def printShowCrosTelemetry(show=False):
     supported_readmask_values.sort()
     supported_readmask_map = {item.lower():item for item in supported_readmask_values}
     i = 3
+    if mode == 'info':
+        filter_ = f'serialNumber={sys.argv[i]}'
```

Comment:
Need to handle user running just `gam info crostelemetry` with no serial.

Reply:
All gam info commands take an object identifier, use gam show to do a broader selection

Reply:
My point is you are doing `sys.argv[i]` (where i == 3) without confirming `len(sys.argv) >= 4`. If the user runs simply `gam info crostelemetry` it only get's caught as an IndexError way up in __init__.py and the user gets a useless generic message. We should confirm user entered a serial number and throw a (nicer) error if not.

I'm sure we don't catch the same thing in other info commands but that doesn't mean we should make the same mistakes here.